packages/dashboard/src/vaadin-dashboard.d.ts

```diff
@@ -97,23 +97,23 @@ export type DashboardItemRemovedEvent<TItem extends DashboardItem> = CustomEvent
  * Fired when an item selected state changed
  */
 export type DashboardItemSelectedChangedEvent<TItem extends DashboardItem> = CustomEvent<{
-  item: TItem;
+  item: TItem | DashboardSectionItem<TItem>;
   value: boolean;
 }>;
 
 /**
  * Fired when an item move mode changed
  */
 export type DashboardItemMoveModeChangedEvent<TItem extends DashboardItem> = CustomEvent<{
-  item: TItem;
+  item: TItem | DashboardSectionItem<TItem>;
   value: boolean;
 }>;
 
 /**
  * Fired when an item resize mode changed
  */
 export type DashboardItemResizeModeChangedEvent<TItem extends DashboardItem> = CustomEvent<{
-  item: TItem;
+  item: TItem | DashboardSectionItem<TItem>;
```

Comment:
A section can't be resized so this should always be a `TItem`

Reply:
Right, reverted the type for that event.